addon/components/o-s-s/scrollable-panel.stories.js

```diff
@@ -68,5 +98,37 @@ const Template = (args) => ({
   context: args
 });
 
+const TemplateHorizontal = (args) => ({
+  template: hbs`
+    <div style="height:70px; width: 300px; background-color: white; " >
+      <OSS::ScrollablePanel @plain={{this.plain}}
+                            @disableShadows={{this.disableShadows}}
+                            @onBottomReached={{this.onBottomReached}}
+                            @hideScrollbar={{this.hideScrollbar}}
+                            @horizontal={{this.horizontal}} >
+        <div class="fx-row fx-gap-px-12 padding-px-12" style="width: fit-content">
+          <div class="background-color-gray-200" style="height: 50px; width: 100px;" />
+          <div class="background-color-gray-200" style="height: 50px; width: 100px;" />
+          <div class="background-color-gray-200" style="height: 50px; width: 100px;" />
+          <div class="background-color-gray-200" style="height: 50px; width: 100px;" />
+          <div class="background-color-gray-200" style="height: 50px; width: 100px;" />
+        </div>
+      </OSS::ScrollablePanel>
+    </div>
+  `,
+
+  context: args
+});
+
 export const BasicUsage = Template.bind({});
 BasicUsage.args = defaultArgs;
+BasicUsage.argTypes = {
+  horizontal: {
+    table: {
+      disable: true
```

Comment:
I think it might be best to leave the option visible in the BasicUsage. I'm afraid we'll completely miss the option when looking into the docs and not checking the sub-usages.